apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -434,13 +431,17 @@ function TokenDetails({ coinType }: TokenDetailsProps) {
 									) : null}
 									<div className="grid grid-cols-3 gap-3 w-full">
 										<LargeButton
+											spacing="sm"
```

Comment:
Now that this is a bigger button it doesn't look right especially with the faucet button there as well for networks other than mainnet